src/classes/UTIL_Namespace_API.cls

```diff
@@ -0,0 +1,78 @@
+/**
+* @author Salesforce.org
+* @date 2018
+* @group API
+* @group-content ../../ApexDocContent/API.htm
+* @description API class that exposes the Namespace Utility methods.
+*/
+global class UTIL_Namespace_API {
+
+    /*******************************************************************************************************
+    * @description Finds the namespace for the current context.
+    * @return string The current namespace as a string, or a blank string if we're not in a namespaced context.
+    */
+    global static String getNamespace() {
+        return UTIL_Namespace.getNamespace();
+    }
+
+    /*******************************************************************************************************
+    * @description Static method adds the current namespace to a string if we're in a managed context,
+    * with double underscore notation. If we're not in a managed context, returns the string itself.
+    * @param str Token name.
+    * @return Token name, with namespace prefix and double underscore if in a managed package.
+    */
+    global static String StrTokenNSPrefix(String str) {
+        return UTIL_Namespace.StrTokenNSPrefix(str);
```

Comment:
The following method will have a dependency with getNamespace() method. Might need to update them.